src/main/java/teammates/common/datatransfer/CommonBundle.java

```diff
@@ -0,0 +1,93 @@
+package teammates.common.datatransfer;
+
+/**
+ * Represents common details of an account.
+ *  * <br> Contains:
+ *  * <br> * account name, email, google id, course name, institution.
+ *  * <br> * link for join course, home page and manage account.
+ */
+public class CommonBundle {
```

Comment:
It was partially my mistake before to give is a vague name such as `CommonBundle`, but at that time this class was confined at the action class. Now that this class is in the DTO package, it needs a better name. A possible name would be `CommonAccountSearchResult`.